src/components/Profilemission.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Profilemission = ({ reservedMission }) => {
+  console.log(reservedMission);
```

Comment:
Please remove this console.log statement,  commits should not contain this because it's against JavaScript best practices